web/modules/custom/lit_search/src/SearchGroup.php

```diff
@@ -28,59 +34,82 @@ class SearchGroup {
    * SearchGroup constructor.
    *
    * @param string $title
+   *   A title.
    */
   public function __construct(string $title) {
     $this->title = $title;
   }
 
   /**
+   * Get method for title.
+   *
    * @return string
+   *   The title
    */
   public function getTitle(): string {
     return $this->title;
   }
 
   /**
+   * Set method for title.
+   *
    * @param string $title
+   *   The title.
+   *
    * @return $this
+   *   The class with title included.
    */
-  public function setTitle(string $title) {
+  public function setTitle(string $title): static {
     $this->title = $title;
 
     return $this;
   }
 
   /**
+   * Get method for count.
+   *
    * @return int
+   *   The count.
    */
   public function getCount(): int {
     return $this->count;
   }
 
   /**
+   * Set method for count.
+   *
    * @param int $count
+   *   The count.
    *
    * @return $this
+   *   The class with count included.
    */
-  public function setCount(int $count) {
+  public function setCount(int $count): static {
```

Comment:
All the set functions in this calls have return type `static` should it be `SearchGroup`

Reply:
Changed static returns.